src/Algolia.Search/Models/Settings/IndexSettings.cs

```diff
@@ -91,6 +92,13 @@ private List<string> AttributesToIndex
         /// </summary>
         public int? RelevancyStrictness { get; set; }
 
+        /// <summary>
+        /// Content defining how the search interface should be rendered. 
+        /// This is set via the settings for a default value and can be overridden via rules.
+        /// </summary>
+        [JsonExtensionData]
```

Comment:
```suggestion
```

I don' think that's useful, is that a glyph? see 86bcb969249c21198b07d540362aa754b97bf5e0